pepper-apis/docs/specification/src/components/schemas.yml

```
@@ -1153,6 +1153,10 @@ Question:
     tooltip:
       type: string
       description: tooltip for the question, in plain text
+    readonly:
+      type: boolean
+      description: false if the question can be modified
+      example: false
```

Given this is a non-nullable, optional property, this needs a default value defined. Since this also changes the default behavior when `true`, It would also more helpful to write the `description` from the perspective of "What changes when this is `true`?" instead of "What does `false` mean?"

```suggestion
    readonly:
      type: boolean
      description: if `true`, the client should not persist any changes to the question's answer. Any requests attempting to update this question's answer will result in an error.
      default: false
```